lib/dns/verifier.js

```diff
@@ -93,9 +93,8 @@ async function verifyRRSet(sig, rrs) {
 	}else{
     // Find the keys that signed this RRSET
     sets = await queryWithProof('DNSKEY', sigdata.signersName);
-    if(sets){
-      keys = sets[sets.length - 1].rrs;
-    }
+    if(!sets){ throw('No sets found on ' + sigdata.signersName)};
```

Comment:
If we write shorthand code like this can you at least remove the curly brackets. ```{}```

Reply:
prettier fixed by itself.